extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcConfigurationMetadata.java

```diff
@@ -74,12 +77,27 @@ public String getEndSessionUri() {
         return endSessionUri;
     }
 
+    public List<String> getSupportedScopes() {
+        return getStringArray(SCOPES_SUPPORTED);
+    }
+
     public String getIssuer() {
         return issuer;
     }
 
     public String get(String propertyName) {
-        return json != null ? null : json.getString(propertyName);
+        return json == null ? null : json.getString(propertyName);
+    }
+
+    public List<String> getStringArray(String propertyName) {
```

Comment:
`getStringList`?

Reply:
@gastaldi, sure (in the json it is an array property - so i started with a method returning `called `getStringArray`, and also returning `Set` but then changed to `List` just in case some duplicates are present, not really necessary - but looks like List is used everywhere to represent the arrays)